src/Controllers/PushController.js

```diff
@@ -84,6 +88,34 @@ export class PushController {
     }
     return expirationTime.valueOf();
   }
+
+  /**
+   * Get push time from the request body.
+   * @param {Object} request A request object
+   * @returns {Number|undefined} The push time if it exists in the request
+   */
+  static getPushTime(body = {}) {
```

Comment:
i wish i knew enough to help.. i will check your test to help further in any future contribution. Tks in advance @flovilmart